### PR TITLE
SUPESC-621 Updated Unzer IG.

### PR DESCRIPTION
**Pull request by romanhavrylko**
## PR Description

Ticket: https://spryker.atlassian.net/browse/SUPESC-621

## Checklist
- [ ] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

**Review by kkicheglovspryker — DISMISSED** [review]

File: docs/pbc/all/payment-service-providers/unzer/install-unzer/integrate-unzer.md [hunk 1]
```diff
@@ -68,10 +77,256 @@ $config[SalesConstants::PAYMENT_METHOD_STATEMACHINE_MAPPING] = [
...
+
+---
+
+### 3) Add translations
+
+Append glossary according to your configuration:
+
+**data/import/common/common/glossary.csv**
+
+```yaml
```
Comment (DmitryLymarenko) [inline docs/pbc/all/payment-service-providers/unzer/install-unzer/integrate-unzer.md:241]: not a yaml

File: docs/pbc/all/payment-service-providers/unzer/install-unzer/integrate-unzer.md [hunk 2]
```diff
@@ -419,33 +675,23 @@ use SprykerEco\Yves\Unzer\Plugin\StepEngine\UnzerStepHandlerPlugin;
...
 
-</details>
+### 1) Set up behavior
 
-11. Add Unzer routes for Yves:
+Set up the following behaviors:
+
+| PLUGIN                   | SPECIFICATION                                | PREREQUISITES | NAMESPACE                                                    |
+|--------------------------|----------------------------------------------|---------------|--------------------------------------------------------------|
+| UnzerRouteProviderPlugin | Adds Unzer module routes to RouteCollection. | None          | SprykerEco\Yves\Unzer\Plugin\Router\UnzerRouteProviderPlugin |
```
Comment (DmitryLymarenko) [inline docs/pbc/all/payment-service-providers/unzer/install-unzer/integrate-unzer.md:694]: namespace is wrong

File: docs/pbc/all/payment-service-providers/unzer/install-unzer/integrate-unzer.md [hunk 3]
```diff
@@ -68,10 +77,256 @@ $config[SalesConstants::PAYMENT_METHOD_STATEMACHINE_MAPPING] = [
...
+---
 
-3. Add the Unzer plugin for `CartDepenencyProvider`:
+### 5) Set up behavior
+
+Set up the following behaviors:
+
+| PLUGIN                                      | SPECIFICATION                                                                                                                                                | PREREQUISITES | NAMESPACE                                                                                        |
+|---------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------|--------------------------------------------------------------------------------------------------|
+| UnzerCredentialsCartOperationPostSavePlugin | Expands `QuoteTransfer` with `UnzerCredentialsTransfer` according to added items.                                                                            | None          | SprykerEco\Zed\Unzer\Communication\Plugin\Cart\UnzerCredentialsCartOperationPostSavePlugin       |
```
Comment (DmitryLymarenko) [inline docs/pbc/all/payment-service-providers/unzer/install-unzer/integrate-unzer.md:302]: please fix data at `namespace` column
namespaces are wrong

File: docs/pbc/all/payment-service-providers/unzer/install-unzer/integrate-unzer.md [hunk 4]
```diff
@@ -352,7 +608,7 @@ class UnzerGuiDependencyProvider extends SprykerUnzerGuiDependencyProvider
 
 </details>
 
-9. Add `StepHandler` plugins to `CheckoutPageDependencyProvider`:
+7. Add `StepHandler` plugins to `CheckoutPageDependencyProvider`:
 
 <details>
 <summary markdown='span'>src/Pyz/Zed/UnzerGui/UnzerGuiDependencyProvider.php</summary>
```
Comment (DmitryLymarenko) [inline docs/pbc/all/payment-service-providers/unzer/install-unzer/integrate-unzer.md:614]: please check this block
path and namespace do not match each other
class definition is missing

**Review by DmitryLymarenko — APPROVED** [review]

**Review by AdvaBD — COMMENTED** [review]
Optional: if the main seller is a merchant, select MERCHANT REFERENCE. > if the operator is a merchant, select MERCHANT REFERENCE.

File: docs/pbc/all/payment-service-providers/unzer/install-unzer/integrate-unzer.md [hunk 5]
```diff
@@ -1,14 +1,20 @@
...
 template: feature-integration-guide-template
 related:
 - title: Install and configure Unzer
   link: docs/pbc/all/payment-service-providers/unzer/install-unzer/install-and-configure-unzer.html
 ---
 
-This document shows how to integrate Unzer into your project.
+# Unzer feature integration
+
+This document describes how to integrate the [Unzer](/docs/pbc/all/payment-service-providers/unzer/unzer.html) into a Spryker project.
```
Comment (AdvaBD) [inline docs/pbc/all/payment-service-providers/unzer/install-unzer/integrate-unzer.md:13]: This document describes how to integrate Unzer > delete the

**Review by AdvaBD — CHANGES_REQUESTED** [review]

**Review by AdvaBD — APPROVED** [review]

**Review by lenadoc — APPROVED** [review]

**Pull request merged**